charts/linkerd2/values.yaml

```diff
@@ -124,3 +124,5 @@ ProxyInjectDisabled: disabled
 ControllerComponentLabel: linkerd.io/control-plane-component
 ControllerNamespaceLabel: linkerd.io/control-plane-ns
 LinkerdNamespaceLabel: linkerd.io/is-control-plane
+
+NodeSelector: {}
```

Comment:
Mind adding a comment here to explain what it is and what it does?

Reply:
Sure, done!